eahub/templates/eahub/edit_messaging_preferences.html

```diff
@@ -0,0 +1,27 @@
+{% extends 'form.html' %}
```

Comment:
Looking at your initial implementation of this with a simple check box on the main profile editing page, I think this is actually enough and we don't need this extra page (sorry for changing my mind here). I would simply add the info that people will not know your email address to the check box.